dashboard/test/factories/factories.rb

```diff
@@ -1832,4 +1832,15 @@
   factory :learning_goal_evaluation do
     association :learning_goal
   end
```

Comment:
in terms of next steps: is there one step after this PR, to remove this and everything else related to `learning_goal_evaluation`? and then one more to remove the table?

Reply:
Yep that was what I was thinking. There are a lot of balls in the air, so I want to make sure nothing gets missed.